ProjetForm/GameModeForm.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace ProjetForm
+{
+    public partial class GameModeForm : Form
+    {
+        public GameModeForm()
+        {
+            InitializeComponent();
+        }
+
+        private void radioButtonGameMode1_CheckedChanged(object sender, EventArgs e)
+        {
+            // Changement d'état des radioButtons
+            radioButtonGameMode2.Checked = false;
+
+            // Affichage (ou non) des groupBox en fonction des radioButtons
+            groupBoxGameModeInput1.Visible = true;
+            groupBoxGameModeInput2.Visible = false;
+        }
+
+        private void radioButtonGameMode2_CheckedChanged(object sender, EventArgs e)
```

Comment:
Uses conventional naming
```suggestion
        private void radioButtonGameMode2_CheckedChanged(object p_Sender, EventArgs p_EventArgs)
```